scripts/release/check_commit_metadata.sh

```diff
@@ -40,7 +40,7 @@ ignore_missing_metadata=${CODER_IGNORE_MISSING_COMMIT_METADATA:-0}
 
 main() {
 	# Match a commit prefix pattern, e.g. feat: or feat(site):.
-	prefix_pattern="^([a-z]+)(\([a-z]*\))?:"
+	prefix_pattern="^([a-z]+)(\([^)]*\))?:"
```

Comment:
I figure `frobulate():` is not a valid prefix, but `frobulate(globular):` is?

```suggestion
	prefix_pattern="^([a-z]+)(\([^)]+\))?:"
```

Reply:
I agree that's preferable, and it seems cdr-bot doesn't like `x():` either, so let's go with that. 😄